apicapi/apic_client.py

```diff
@@ -542,10 +542,11 @@ def _send(self, request, url, data=None, refreshed=None, accepted=None,
               sleep_offset=0):
         """Send a request and process the response."""
         curr_call = time.time()
+        print(self.sleep + sleep_offset, curr_call - self.last_call)
         try:
             time.sleep((self.sleep + sleep_offset) -
                        (curr_call - self.last_call))
-        except IOError:
+        except (IOError, ValueError):
```

Comment:
time.sleep() now raises a `ValueError` if the sleep length is
negative, instead of an infinite sleep on Windows or raising an IOError on
Linux for example, to have the same behaviour on all platforms.

https://hg.python.org/cpython/rev/0e5485634817

Reply:
@kentwu1 If you agree with me on this, can you resolve conversation & approve this change?